pkg/graveler/graveler.go

```diff
@@ -2014,15 +2065,13 @@ func (g *KVGraveler) sealedEmpty(ctx context.Context, repositoryID RepositoryID,
 }
 
 // dropStaging deletes all staging area entries of a given branch from store
```

Comment:
```suggestion
// dropTokens deletes all staging area entries of a given branch from store
```

Reply:
Done.

Reply:
think it is still relevant - I see the comment is 'dropStaging'